File: main/hw2/python/hello-json.py
#!/usr/bin/env python3
import json
import os
import datetime

# Headers
print("Cache-Control: no-cache")
print("Content-type: application/json\n")

# Data Dictionary
data = {
    "message": "Hello JSON World from Python",
    "date": str(datetime.datetime.now()),
    "ip_address": os.environ.get('REMOTE_ADDR', 'Unknown')
}

# Print JSON
print(json.dumps(data))

File: main/hw2/ruby/hello-json.rb
#!/usr/bin/ruby
require 'json'

puts "Cache-Control: no-cache"
puts "Content-type: application/json\n\n"

data = {
  "message" => "Hello JSON World from Ruby",
  "date" => Time.now.strftime('%Y-%m-%d %H:%M:%S'),
  "ip_address" => ENV['REMOTE_ADDR']
}

puts JSON.generate(data)